static/app/views/settings/organizationAuditLog/index.tsx

```diff
@@ -72,7 +72,7 @@ function OrganizationAuditLog({location}: Props) {
       setState(prevState => ({
         ...prevState,
         entryList: data.rows,
-        eventTypes: data.options.sort(),
+        eventTypes: data.options,
```

Comment:
Why are we removing the sort here?

Reply:
I moved the sort into the other file since we need to sort _after_ replacing the entry types.

https://github.com/getsentry/sentry/blob/c876ace5b786784015d9bc3888f248344ede40f8/static/app/views/settings/organizationAuditLog/auditLogList.tsx#L68-L92C17